Lib/test/test_capi.py

```diff
@@ -801,6 +801,9 @@ class Test_testcapi(unittest.TestCase):
     def test_widechar(self):
         _testcapi.test_widechar()
 
+    def test_version_api_data(self):
+        assert _testcapi.Py_Version == sys.hexversion
```

Comment:
We use unittest, not pytest, you should write:

```suggestion
        self.assertEqual(_testcapi.Py_Version, sys.hexversion)
```